test/backend_pool.in.cpp

```diff
@@ -1235,3 +1235,84 @@ NGRAPH_TEST_P(${BACKEND_NAME}, avg_pool_3d_params, avg_pool_3d_uneven_strided_pa
 
 // avg_pool_3d case generation
 NGRAPH_INSTANTIATE_TEST_CASE_P(${BACKEND_NAME}, include_pad, avg_pool_3d_params, testing::Bool());
+
```

Comment:
@aprocter, I added a description of the test as suggested and some comments through out. Feel free to poke at it if it isn't quite clear.